packages/components/src/internal/components/samples/SampleDetailEditing.tsx

```diff
@@ -101,6 +108,20 @@ export class SampleDetailEditing extends PureComponent<Props, State> {
         return getGroupedSampleDisplayColumns(detailColumns, updateColumns, sampleTypeDomainFields, isAliquot);
     };
 
+    getAliquotRootSampleQueryConfig = () : QueryConfig => {
```

Comment:
Maybe you are already planning to do this during when writing the jest tests, but this might be easier to unit test if it was factored out as a util function that takes in the `row` and `sampleSet` as props and returns the `QueryConfig`.